addons/l10n_in_pos/static/src/overrides/models/pos_order.js

```diff
@@ -37,7 +37,9 @@ patch(PosOrder.prototype, {
                 l10n_in_hsn_code: hsnCode,
                 price_unit: priceUnit,
                 quantity: line.get_quantity(),
+                discount: line.get_discount(),
                 uom: null,
+                taxes: taxes,
                 ...getTaxesValues(
```

Comment:
now remove the call to `getTaxesValues`. However, this has nothing to do with the current forward-port

Reply:
Removed it, can you please check it again.